root/lib/systemd/system/titus-salt-generator.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
Another @gabrielhartmann question. Do we need to explicitly enable this, or will the first time the titus-metatdataproxy is started start this?

Reply:
From experimentation, it runs the first time the proxy is started.